packages/jumpstarter/jumpstarter/common/utils_test.py

```diff
@@ -0,0 +1,21 @@
+from .utils import launch_shell
+
+
+def test_launch_shell(tmp_path, monkeypatch):
+    monkeypatch.setenv("SHELL", "/bin/true")
+    exit_code = launch_shell(
+        host=str(tmp_path / "test.sock"),
+        context="remote",
+        allow=["*"],
+        unsafe=False
+    )
+    assert exit_code == 0
+
+    monkeypatch.setenv("SHELL", "/bin/false")
```

Comment:
```suggestion
    monkeypatch.setenv("SHELL", shutil.which("false"))
```

`/bin/false` is not always present.